### PR TITLE
New operations for adhealth

### PR DESCRIPTION
**Pull request by binilkk**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [X] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
Lint check got stuck, have restarted

### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/examples/ConnectorMetadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "api-version": "2014-01-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "connectors": [
+              {
+                "connectorId": "Id of the connector",
```
Comment (antmarti-microsoft) [inline specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/examples/ConnectorMetadata.json]: Can you give an example of this field which looks like a real-life connectorId? Is this an ARM resourceId?

File: specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json [hunk 2]
```diff
@@ -4237,6 +4279,40 @@
...
+          "description": "The list of run profile names.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ConnectorMetadataDetails": {
+      "description": "Details of the connector. Only has ConnectorId & Display Name.",
```
Comment (antmarti-microsoft) [inline specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json]: [nit] The schema tells us which properties are in this object - it is superfluous in the description.

File: specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json [hunk 3]
```diff
@@ -3328,6 +3328,48 @@
           "nextLinkName": null
         }
       }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/servicemembers/{serviceMemberId}/metrics/connectormetadata":
```
Comment (antmarti-microsoft) [inline specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json]: Shouldn't {metricName} param be part of this URI?

File: specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json [hunk 4]
```diff
@@ -3328,6 +3328,48 @@
...
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "serviceMemberId",
+            "in": "path",
+            "description": "The server id.",
```
Comment (antmarti-microsoft) [inline specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json]: Is this description correct?

**Review by anthony-c-martin — CHANGES_REQUESTED** [review]

File: specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/examples/ConnectorMetadata.json (hunk 1 above)
Comment (binilkk) [reply specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/examples/ConnectorMetadata.json]: It is not an ARM resource ID, it is just a guid that identifies a AAD Connect Sync connector.

Comment (binilkk) [reply specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/examples/ConnectorMetadata.json]: Here is an example response.   "connectors": [
    {
      "connectorId": "3efbb72d-5047-4816-a7dd-73fefe16a2f3",
      "connectorDisplayName": "adhsdfus1runner.adhs.com"
    },
    {
      "connectorId": "b891884f-051e-4a83-95af-2544101c9083",
      "connectorDisplayName": "dfadhybridhealth.ccsctp.net - AAD"
    }
  ],
  "runProfileNames": [
    "Export",
    "ApplyRules",
    "DeltaImport"
  ]

File: specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json (hunk 2 above)
Comment (binilkk) [reply specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json]: Fixed

File: specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json (hunk 3 above)
Comment (binilkk) [reply specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json]: Fixed.

File: specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json (hunk 4 above)
Comment (binilkk) [reply specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json]: Fixed

File: specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json [hunk 5]
```diff
@@ -3328,6 +3328,55 @@
...
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "serviceMemberId",
+            "in": "path",
+            "description": "The servic member id.",
```
Comment (nschonni) [inline specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json]: servic -> service

Comment (binilkk) [reply specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json]: Fixed.

File: specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/examples/ConnectorMetadata.json (hunk 1 above)
Comment (antmarti-microsoft) [reply specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/examples/ConnectorMetadata.json]: Thanks! I meant to ask - can you update the PR with this? It's always useful if the examples show data which looks like real-life data.

**Review by anthony-c-martin — APPROVED** [review]

**Review by praries880 — APPROVED** [review]

**Review by praries880 — APPROVED** [review]

**Comment by praries880** [comment]
closed and re-opened the PR to kick off CI again

**Pull request merged**